client/src/pages/donate.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable max-len */
+import React, { Component, Fragment } from 'react';
+import Helmet from 'react-helmet';
+import { StripeProvider, Elements } from 'react-stripe-elements';
+import { Row, Col } from '@freecodecamp/react-bootstrap';
+
+import { stripePublicKey } from '../../config/env.json';
+
+import Spacer from '../components/helpers/Spacer';
+import DonateForm from '../components/Donation/components/DonateForm';
+import DonateText from '../components/Donation/components/DonateText';
+import PoweredByStripe from '../components/Donation/components/poweredByStripe';
+
+import './index.css';
```

Comment:
Is this still correct?

Reply:
Yes it is. 

All pages are deriving styles from this common `index.css`? Unless you want to rename and re-include in all pages? Oversight from the refactor? If yes, then probably it should come in from your PR planned later?